### ISSUE TITLE
Titles with multiple languages/scripts in IIIF view

### ISSUE THREAD

**Issue by escowles** [pulibrary/marc_liberation#76]
Following on from #74, we should handle titles with multiple languages/scripts.  Looking at about two dozen sample records, I see three patterns:

1. `title_vern_display` = title in native script (with creator and sometimes a translation appended), `title_citation_display`  = transliterated title.  `language_code_s` = language of the title.  The script of the `title_vern_display` can be guessed from the title value or `language_code_s`.

2. `title_vern_display` not present, `language_code_s` = `mul`.  `title_citation_display` = title in native language.  First value in `language_facet` = `Multiple`, second value = language of the title.

3. `title_vern_display` not present, `language_code_s` = language of the title.

**Comment by tampakis** [comment]
`title_display` should be used for the transliterated title instead of `title_citation_display`. The logic looks correct.

**Comment by escowles** [comment]
@jpstroop The [spec](http://iiif.io/api/presentation/2.1/) shows only single, simple titles, though it does say there can be multiple titles.  To do a single title with a language tag, would we do:

```
"label":{"@value":"Ceci n'est pas une tître", "@language":"fre"}
```

And for multiple titles, I assume you would have an array of the title objects like that.

**Comment by escowles** [comment]
@tampakis: But `title_display` (and all of the other titles except `title_citation_display`) have the creator appended to them.

**Comment by tampakis** [comment]
Good point. `title_vern_display` also has creator appended to it. If we don't need want to include the vernacular title we can just use `title_citation_display`.

**Comment by escowles** [comment]
Maybe we could add a version of the vernacular title without the creator appended?

**Comment by tampakis** [comment]
Sure, do we want to add vernacular as a potential 2nd value in the `title_citation_display` field or make a new field for it?

**Comment by escowles** [comment]
Adding it to `title_citation_display` sounds good -- following the pattern of several other fields.

**Comment by tampakis** [comment]
Ok, just remove the `alternate_script` and `first` options from this line: https://github.com/pulibrary/marc_liberation/blob/development/lib/traject_config.rb#L122

**Comment by jpstroop** [comment]
If this "title" is going in `label`, I'd say keep it simple and just use the vernacular title as a string.

**Comment by jpstroop** [comment]
Bah...but even with one title, if it's meant to be rendered rtl, I guess you still need the lang/script.

**Comment by escowles** [comment]
Right, so maybe we want to put the romanized title in `label` and put the vernacular title in the metadata array so we can include the lang/script?

**Comment by jpstroop** [comment]
I guess, though displaying the real, transcribed, title would be ideal. I guess it comes down to being idealistic vs. what clients will support now. Since we're in development, I'm tending toward idealistic for now.

**Comment by escowles** [comment]
Well, we can put a value/language object in label and see what the viewers do with it -- I think that's the "right" thing to do.

If that doesn't work, we can put the real title in label, and put the transliterated title in metadata/alternate with e.g., language=ar-Latn.

**Comment by escowles** [comment]
Closed with #84
